xbmc/music/MusicDatabase.cpp

```diff
@@ -3934,11 +3896,15 @@ void CMusicDatabase::UpdateTables(int version)
       m_pDS->exec("DROP TABLE album");
       m_pDS->exec("ALTER TABLE album_new RENAME TO album");
    }
+   if (version < 55)
```

Comment:
This should be versio < 56 and therfore align with GetSchameVersion. Otherwise people going from 55 to 56 won't get this dropped. Small issue, but yeah.
Or we bump both to 57